test/plugins/validation/2and3/security-definitions-ibm.js

```diff
@@ -115,13 +116,43 @@ describe('validation plugin - semantic - security-definitions-ibm', function() {
         }
       };
 
-      const res = validate({ jsSpec: spec }, config);
+      const res = validate({ resolvedSpec: spec }, config);
       expect(res.errors.length).toEqual(0);
       expect(res.warnings.length).toEqual(0);
     });
   });
 
   describe('OpenAPI 3', function() {
+    it('should catch errors in referenced security definitions', async function() {
+      const spec = {
+        openapi: '3.0.0',
+        info: {
+          version: '3.0.0',
+          title: 'example title'
+        },
+        paths: {},
+        components: {
+          securitySchemes: {
+            SecuritySchemaN1: {
+              description: 'example description',
+              type: 'apiKey',
+              in: 'cookie',
+              name: 'userAuthCookie'
+            }
+          }
+        },
+        securityScheme: {
+          basicAuth: {
+            $ref: '#/components/securitySchemes/SecuritySchemaN1'
+          }
+        }
+      };
+      const resolvedSpec = await resolver.dereference(spec);
+
+      const res = validate({ resolvedSpec }, config);
+      expect(res.errors.length).toEqual(0);
+      expect(res.warnings.length).toEqual(0);
```

Comment:
The intention for this test is to verify that problems are caught in the security definitions validator, so I would adjust the Security Scheme so that it's bad 😈 

Or you can leave this as a positive test and add a new negative test